Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs

```diff
@@ -72,7 +85,7 @@ public IBrowserConfiguration CreateBrowserConfiguration ()
     public ITestInfrastructureConfiguration CreateTestInfrastructureConfiguration ()
     {
       var configSettings = WebTestConfigurationSection.Current;
-      
+

```

Comment:
This line change seems unnecessary, although i am confused because its marked as deleted in the parent version? Maybe the line ending character changed?
![image](https://user-images.githubusercontent.com/13847286/65492651-96144800-deb1-11e9-9aca-220150ea1545.png)